ocs_ci/ocs/resources/pod.py

```diff
@@ -2369,3 +2369,15 @@ def get_pod_ip(pod_obj):
 
     """
     return pod_obj.get().get("status").get("podIP")
+
+
+def wait_for_osd_pods_having_ids(osd_ids, timeout=180, sleep=10):
```

Comment:
Yes, I will add.

Reply:
Why do we always need sleep along with timeout?

Reply:
In case we want to change the time to wait between iterations. I think there must be cases we wish to change this value, so I also added it here. Usually, 10 seconds should be fine.